src/content/en/updates/2019/06/layoutNG.md

```diff
@@ -0,0 +1,328 @@
+
+<div class="w-figures-2-up">
+  <figure>
+    <img class="w-screenshot" loading="lazy" 
+      src="/web/updates/images/2019/06/legacy_float_margin.png" 
+      alt="top text line shown overlaying floated image"
+    >
+    <figcaption>
+      <small>Fig 1a, Legacy layout engine</small><br>
+      Text overlaps the floating image to the right
+    </figcaption>
+  </figure>
+
+  <figure>
+    <img class="w-screenshot" loading="lazy" 
+      src="/web/updates/images/2019/06/ng_float_margin.png" 
+      alt="proper text on left and floated image on right"
+    >
+    <figcaption>
+      <small>Fig 1b, LayoutNG</small><br>
+      Text is positioned beside the floating image to the right
+    </figcaption>
+  </figure>
+</div>
+
+The same problem may occur within a single line. The example below shows 
+a block element with a negative margin following a floating element 
+([#895962](https://crbug.com/895962)). The text should not overlap with the float.
+
+<div class="w-figures-2-up">
+  <figure>
+    <img loading="lazy" 
+      src="/web/updates/images/2019/06/legacy_float_overlap.png" 
+      alt="text line shown overlaying an orange box"
+    >
+    <figcaption>
+      <small>Fig 2a, Legacy layout engine</small><br>
+      Text overlaps the floating orange element
+    </figcaption>
+  </figure>
+
+  <figure>
+    <img loading="lazy" 
+      src="/web/updates/images/2019/06/ng_float_overlap.png" 
+      alt="proper text on right of orange box"
+    >
+    <figcaption>
+      <small>Fig 2b, LayoutNG</small><br>
+      Text is positioned beside the floating orange element
+    </figcaption>
+  </figure>
+</div>
+
+#### Formatting Context Positioning
+When an element forming a block formatting context is sized next to floats, 
+the legacy layout engine would try to size the block a fixed number 
+of times before giving up. This approach led to unpredictable and unstable 
+behavior and didn't match other implementations. In LayoutNG all 
+floats are taken into account when sizing the block. 
+(See [Chromium bug #548033](https://crbug.com/548033).)
+
+Absolute and fixed positioning are now more compliant with W3C specifications 
+and better match the behavior in other browsers. The differences 
+between the two are most visible in two cases:
+
+- **Multi-line inline containing blocks** <br>If an absolutely positioned 
+containing block spanned multiple lines, the legacy engine might 
+incorrectly use only a subset of the lines to compute the containing block bounds.
+- **Vertical writing modes** <br>The legacy engine had many problems placing 
+out-of-flow elements in the default position in vertical writing modes. 
+See the next section for more details about improved writing mode support.
+
+## Right-to-left (RTL) languages and vertical writing modes
+LayoutNG was designed from the ground up to support vertical writing modes 
+and RTL languages, including bidirectional content.
+
+#### Bidirectional Text
```

Comment:
Nit: Should be H3, when the TOC is generated, it'll be treated that way, so it's best to use that.

Reply:
👍